trailblazer/store/models.py

```diff
@@ -130,6 +131,7 @@ class Job(Model):
 
     id = Column(types.Integer, primary_key=True)
     analysis_id = Column(ForeignKey(Analysis.id, ondelete="CASCADE"), nullable=False)
+    upload_analysis_id = Column(ForeignKey(Analysis.id))
```

Comment:
Yes, the plan is to do all uploads via slurm. I think the ones to caesar already are submitted as slurm jobs.

Apparently, it is necessary to define a separate foreign key https://docs.sqlalchemy.org/en/20/orm/join_conditions.html#handling-multiple-join-paths

Reply:
Great, that will be a big improvement! The intent of the field is still a bit unclear to me, i.e how will this fulfill the Acceptance criteria outlined?

Reply:
This PR is just a work item associated with the story and is a step in the implementation described. It will not implement the entire story or fulfill any of the acceptance criteria on its own.

I've decided to change the implementation though, I think it makes more sense to have a single `jobs` field on the analysis and just add a `job_type` field

Reply:
You can see the implementation plan here: https://github.com/Clinical-Genomics/cigrid-ui/issues/492#issuecomment-1905771598

Reply:
Ok, sorry, It's sometimes hard to get the overall picture when there is no usage of the added field. Perhaps the scope of the PR could encompass that as well? if it does not get to bloated.